Winium/Winium.StoreApps.Driver/CommandLineOptions.cs

```diff
@@ -33,6 +33,9 @@ internal class CommandLineOptions
         [Option("nodeconfig", Required = false, HelpText = "configuration JSON file to register driver with selenium grid")]
         public string NodeConfig { get; set; }
 
+        [Option("dependency", Required = false, HelpText = "dependencies to be installed before main app")]
+        public string Dependency { get; set; }
```

Comment:
This command line is not used. Do you want to be able to set dependencies as command line option besides setting it in Capabilities?